holopy/inference/result.py

```diff
@@ -173,22 +184,35 @@ def _save(self, filename, **kwargs):
         ds = self._serialization_ds()
         ds.to_netcdf(filename, engine='h5netcdf', **kwargs)
 
+    # deprecated methods as of 3.3
+    def best_fit(self):
+        # this method is published in the HoloPy paper
+        from holopy.fitting import fit_warning
+        fit_warning('FitResult.inferred_hologram', 'SamplingResult.best_fit()')
+        return self.inferred_hologram
+
+    def output_scatterer(self):
+        from holopy.fitting import fit_warning
+        fit_warning('FitResult.inferred_scatterer',
+                    'SamplingResult.output_scatterer()')
+        return self.inferred_hologram
```

Comment:
Shouldn't this return ```SamplingResult.output_scatterer``` rather than ```self.inferred_hologram```?

If so, can we capture this with a test?

Reply:
Good catch!